tests/app/helpers/conftest.py

```diff
@@ -1,12 +1,194 @@
-from flask.app import Flask
 from pytest import fixture
 
-from app.setup import create_app
+from app.helpers.template_helpers import ContextHelper
 
 
 @fixture
-def app() -> Flask:
-    setting_overrides = {"LOGIN_DISABLED": True}
-    _app = create_app(setting_overrides=setting_overrides)
+def get_context_helper():
+    def _context_helper(
+        app, survey_config, is_post_submission=False, include_csrf_token=True
+    ):
+        with app.test_client():
+            return ContextHelper(
+                language="en",
+                is_post_submission=is_post_submission,
+                include_csrf_token=include_csrf_token,
+                survey_config=survey_config,
+            )
 
-    return _app
+    return _context_helper
+
+
+@fixture(name="footer_context")
+def footer():
+    return {
+        "lang": "en",
+        "crest": True,
+        "newTabWarning": "The following links open in a new tab",
+        "copyrightDeclaration": {
+            "copyright": "Crown copyright and database rights 2020 OS 100019153.",
+            "text": "Use of address data is subject to the terms and conditions.",
+        },
+    }
+
+
+@fixture
+def expected_footer_census_theme(footer_context):
```

Comment:
Personally I'm not a fan of these living here cause it not very clear from the test body, I would prefer if it was in the file it was tests as it's not common to all helpers. Would be nice if it was in its own conftest in its own folder. Not a blocker, don't have a perfect suggestion.

Reply:
I prefer having data separate from tests. This keeps tests clean and readable. Having 100 lines of json in tests makes it quite noisy and imagine scrolling up and down these tests. Keep them in fixture reduced no of lines which are common across all json's. Happy to bring it back if anyone else agrees to it

Reply:
I agree with that, I wasn't that it should be inline in the test body, it was more than it prob should be in a related file. i.e this conftest is for all helpers, but these snippets are only relevant to template helper tests.

Reply:
Keeping in fixture means it loads it automatically without needing any imports (if we keep it in separate file)